app/models/work_original_creator.rb

```diff
@@ -0,0 +1,16 @@
+class WorkOriginalCreator < ApplicationRecord
+  belongs_to :work
+
+  # Get the id and username (if still available) for the associated user.
+  def display
+    user = User.find_by(id: user_id)
```

Comment:
Not really, I just wasn't sure how it would work for deleted users. After some local testing to satisfy my concern I've updated, and it's a lot nicer now